### PR TITLE
Initialise Nucleoid Chat Page

### PR DESCRIPTION
**Pull request by durulkoca**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "eslint-plugin-cypress": "^2.15.1",
     "moment": "^2.29.4",
     "prettier": "^2.8.7",
+    "prismjs": "^1.29.0",
```
Comment (canmingir) [inline package.json:36]: Can you remove this if we don't use?

File: src/App.jsx [hunk 2]
```diff
@@ -154,7 +156,7 @@ function App() {
             return setContext("error");
           });
       } else {
-        window.location.assign(`${window.location.origin}/ide/sample/api`);
+        // window.location.assign(`${window.location.origin}/ide/sample/api`);
```
Comment (canmingir) [inline src/App.jsx:159]: Can you remove this?

File: src/widgets/ChatMainArea/ChatDisplay.jsx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+import React, { useEffect, useState } from "react";
+
+const ChatDisplay = ({ chat }) => {
+  const theme = useTheme();
+  const [isLoading, setIsLoading] = useState(false);
+  const [openDialog, setOpenDialog] = useState(false);
+  const [selectedCode, setSelectedCode] = useState("");
+
+  useEffect(() => {
+    setIsLoading(true);
```
Comment (canmingir) [inline src/widgets/ChatMainArea/ChatDisplay.jsx:22]: Can you add TODO comment if this is temporarily?

File: src/widgets/ChatMainArea/MonacoEditorWrapper.jsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import MonacoEditor from "@monaco-editor/react";
+import monacoDarkTheme from "../../lib/monacoEditorTheme.json";
+import { useStorage } from "@nucleoidjs/webstorage";
+import { v4 as uuidv4 } from "uuid";
+
+import React, { useEffect, useState } from "react";
+
+const MonacoEditorWrapper = ({ code, readOnly }) => {
```
Comment (canmingir) [inline src/widgets/ChatMainArea/MonacoEditorWrapper.jsx:8]: Can you rename to chat `ChatEditor`?

File: package.json (hunk 1 above)
Comment (durulkoca) [reply package.json:36]: removed unused library

**Review by canmingir — APPROVED** [review]

**Pull request merged**